server/controllers/disk.coffee

```diff
@@ -85,7 +85,9 @@ module.exports.info = (req, res, next) ->
         for line in lines
             line = line.replace /[\s]+/g, ' '
             lineData = line.split(' ')
-            if lineData.length > 5 and (lineData[5] is '/' or dir.indexOf(lineData[5]) isnt -1)
+            if lineData.length > 5 and (lineData[5] is '/' or
+            (dir.indexOf(lineData[5]) isnt -1) or
+            (lineData[5].indexOf(dir) isnt -1))
```

Comment:
Pourquoi `isnt 0` alors que plus bas, on teste justement `(dir.indexOf(lineData[5]) is 0) or (lineData[5].indexOf(dir) is 0)` ?